packages/webapp/src/routes/layout.test.ts

```
@@ -100,17 +99,23 @@ describe('Layout component', () => {
-	it('renders Homepage when on root path', async () => {
-		mockPageStore.mockSetSubscribeValue({
-			...initialPageState,
-			url: new URL('http://localhost/')
-		});
-
-		const { container } = render(Layout);
-
-		expect(container.querySelector('main')).not.toBeInTheDocument();
-		expect(screen.getByTestId('homepage')).toBeInTheDocument();
-	});
+	it.only('renders Homepage when on root path', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)